src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -14,6 +15,11 @@
 
 log = logging.getLogger(__name__)
 
+@unique
+class SubvolumeFeatures(Enum):
+    FEATURE_CLONE       = "clone"
```

Comment:
Do you see a benefit in adding more detail to the keys and values,
e.g.,
       FEATURE_SNAPSHOT_CLONE = "snapshot clone"
       FEATURE_SNAPSHOT_AUTOPROTECT="snapshot auto-protect"

Reply:
Hmmm... feature "strings" should be long lasting IMO, so making them more detailed helps retain meaning for a longer duration and also avoid potential conflicts. I do not prefer spaces in feature strings, hence will use "snapshot-clone" and "snapshot-autoprotect" and amend this PR.

Reply:
Addressed in the latest update.

Reply:
Thanks!